### ISSUE TITLE
PY_SSIZE_T_CLEAN error on compilation

### ISSUE THREAD

**Issue by rootabeta** [TheWover/donut#107]
```python
>>> import donut
>>> shellcode = donut.create('/home/rootabeta/Projects/PoCDLL/evil.exe',entropy=1,format=1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
SystemError: PY_SSIZE_T_CLEAN macro must be defined for '#' formats
```
Python 3.10.2 on 5.16.15-arch1-1

evil.exe source:
```c++
#include <windows.h>
#include <tchar.h>

int main() {
	MessageBox(NULL, "BOO!", "Get spooky!", MB_OK | MB_SYSTEMMODAL);

    return 0;  
}
```
compiled with 
x86_64-w64-mingw32-g++ exe.cc -o evil.exe

**Comment by S4ntiagoP** [comment]
This should be resolved in this PR: https://github.com/TheWover/donut/pull/101

**Comment by TheWover** [comment]
Fixed in `dev` branch for v1.0
